DisguiseUnityRenderStream/Editor/DisguiseRenderStreamSettingsEditor.uxml

```diff
@@ -0,0 +1,4 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements">
+    <uie:PropertyField label="Scene Control" binding-path="sceneControl" />
```

Comment:
Minor: In code, I've started using [`L10.Tr()`](https://docs.unity3d.com/ScriptReference/L10n.Tr.html) for strings, because this was the recommendation I got on the UIToolkit slack. Unfortunately, they don't have a good answer for strings UXML. Figures 😞